core-primitives/substrate-sgx/externalities/src/lib.rs

```diff
@@ -102,6 +103,16 @@ impl SgxExternalitiesTrait for SgxExternalities {
 		self.state.contains_key(k)
 	}
 
+	/// get the next key in state after the given one (excluded) in lexicographic order
+	fn next_storage_key(&self, key: &[u8]) -> Option<Vec<u8>> {
+		use std::ops::Bound;
```

Comment:
Could you move this to the top of the file? It helps to have the imports in one place to have an overview.

Reply:
Thanks, will do